src/ai/backend/client/cli/vfolder.py

```diff
@@ -441,3 +441,27 @@ def invitations():
         except Exception as e:
             print_error(e)
             sys.exit(1)
+
+
+@vfolder.command()
+@click.argument('name', type=str)
+def leave(name):
+    '''Leave the shared virutal folder.
+
+    NAME: Name of a virtual folder
+    '''
+    with Session() as session:
+        try:
+            vfolder_info = session.VFolder(name).info()
+            if vfolder_info['type'] == 'group':
+                print('You cannot leave a group virtual folder.')
+                return
+            if vfolder_info['is_owner']:
+                print('You cannot leave a virtual folder you own. Consider using delete instead.')
+                return
```

Comment:
Yes, it is good to have validations on both the server-side (for correctness) and client-side (for UX). :+1: